examples/simple_query_in_go/main.go

```diff
@@ -0,0 +1,87 @@
+/*  Cardano on BigQuery
+
+    example: simple query
+*/
+
+
+/*
+authenticate: `gcloud auth application-default login`
+prepare: `go mod tidy -v`
+compile: `go build`
+run: `go run . --epoch_no 321`
+ or: `./simple_query -epoch_no 321`
+*/
```

Comment:
would it be useful to mention here that the script retrieves the last block of the epoch that gets as a parameter? Or extract this into a README? 🤷

Reply:
I am stuck at the step `gcloud auth application-default login` ..